CONTRIBUTING.md

```diff
@@ -72,6 +72,10 @@ Packetbeat:
 Some of the Beats might have extra development requirements, in which case you'll find a
 CONTRIBUTING.md file in the Beat directory.
 
+We use an [EditorConfig](http://editorconfig.org/) file in the beats repository to standardise 
+how different editors use whitespaces etc. in our files. Most popular editors have a
```

Comment:
Avoid using "etc" because it can sometimes be a problem when the doc gets translated. How about being more specific?

...how different editors handle whitespace, tabs, and other coding styles in our files...